### PR TITLE
Add donut2 to CI map testing [MDB IGNORE]

### PR DESCRIPTION
**Pull request by pgmzeta**
[TOOLING][CODE QUALITY]
## About the PR 
Adds donut2 to CI map testing

## Why's this needed? 
donut2 is in rotation, so let's make sure it's not breaking

### CODE REVIEW

**Comment by pgmzeta** [comment]
donut2 needs some fixin' before this can go in. drafting.

File: .github/workflows/automaton.yml [hunk 1]
```diff
@@ -90,7 +90,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        map: ["COGMAP", "COGMAP2", "CLARION", "DESTINY", "DONUT3", "OSHAN", "KONDARU", "ATLAS", "SPACE_PREFAB", "UNDERWATER_PREFAB", "FULL"]
+        map: ["COGMAP", "COGMAP2", "CLARION", "DESTINY", "DONUT3", "DONUT2", "OSHAN", "KONDARU", "ATLAS", "MANTA", "SPACE_PREFAB", "UNDERWATER_PREFAB", "FULL"]
```
Comment (ZeWaka) [inline .github/workflows/automaton.yml:93]: nice try

**Review by ZeWaka — CHANGES_REQUESTED** [review]
donut2 should also be moved from unused

File: .github/workflows/automaton.yml (hunk 1 above)
Comment (pgmzeta) [reply .github/workflows/automaton.yml:93]: my PR was gonna handle this all but nooooo >:( 

I'll fix the merge issue :)

Comment (pgmzeta) [reply .github/workflows/automaton.yml]: should be fixed in [5c86fc2](https://github.com/goonstation/goonstation/pull/10285/commits/5c86fc2d85672abed8a25c6fe20fba1e486d4add)

**Comment by pgmzeta** [comment]
merge conflict ???

**Pull request merged**